### ISSUE TITLE
CI tests are failing (ci-005 and hotfix-015)

### ISSUE THREAD

**Issue by mfry-ll** [iqcToolbox/iqcToolbox#76]
The "All iqcToolbox tests" github action is [failing](https://github.com/iqcToolbox/iqcToolbox/actions/runs/3347018342).  However, these tests pass when executing the "Multple OS/MATLAB" github [action ](https://github.com/iqcToolbox/iqcToolbox/actions/runs/3353458005)for ubuntu18.04, ubuntu 20.04, and MATLAB R2020a - R2021b.  Also, these tests pass under a local fresh install of the toolbox (Windows, R2022a). MATLAB R2022b was recently released for github actions.

**Comment by mfry-ll** [comment]
Closed by #77

**Comment by mfry-ll** [comment]
This issue is not related to MATLAB Github Actions interacting with R2022b.  I get the same failures when testing on my local installation of R2022b. For example:
```
tests.run_quick_tests
```
produces this snippet:
```
================================================================================
Error occurred in tests.iqc_analysis.testIqcAnalysisDlti/testSmallGainRobustlyStableSystem and it did not run to completion.
    ---------
    Error ID:
    ---------
    'Control:ltiobject:empty'
    --------------
    Error Details:
    --------------
    Error using DeltaDlti/deltaToMultiplier
    Error defining property 'basis_function' of class 'MultiplierDlti'. Not supported for model
    objects.
    
    Error in iqcAnalysis (line 152)
            mults_del(i) = deltaToMultiplier(delta.deltas{i},...
    
    Error in tests.iqc_analysis.testIqcAnalysisDlti/testSmallGainRobustlyStableSystem (line 59)
        [result, valid] = iqcAnalysis(lft, 'analysis_options', opts);
================================================================================
```
Will need to investigate what changes in R2022b in the Control System Toolbox cause this new failure.
